### PR TITLE
Morgan quarantine updates

### PR DESCRIPTION
**Pull request by iamdamosuzuki**
I added an article called "level clamping" and some associated image files.

### CODE REVIEW

File: _artifacts/level_clamping.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+Once a video signal is clamped or legalized in a signal chain the lost information can not be regained. This is why it is very important to make sure that no items in your signal chain are legalizing the signal before digitization.
+
+## Notes regarding Station Qualification
+
+As mentioned above, any information lost due to clamping or legalizing cannot be regained. This is why this artifact should be checked for during Video Digitization Station Qualification. You should ensure that any potential signal chain and any video equipment being used (TBCs, Frame Synchronizers, Switches, Digital I/Os, Digitizers, etc) do not clamp or legalize the video signal. This can be easily checked using QCTools to view the waveforms. Below are examples that show this clearly.
+
+## Example(s)
+
+Below is what SMPTE bars look like normally. Not how there is video signal above 100 IRE and below 0 IRE
```
Comment (kieranjol) [inline _artifacts/level_clamping.md:35]: Note

Comment (iamdamosuzuki) [reply _artifacts/level_clamping.md]: fixed!

**Comment by ablwr** [comment]
Need a rebase to reflect the Gemfile changes but otherwise LGTM!

**Comment by iamdamosuzuki** [comment]
I fixed the Gemfile issue. thanks!

**Pull request merged**